packages/core/__tests__/Platform-test.ts

```diff
@@ -81,3 +81,29 @@ describe('Platform test', () => {
 		});
 	});
 });
+
+describe('detectFramework observers', () => {
+	let module;
+
+	beforeAll(() => {
+		jest.resetModules();
+		module = require('../src/Platform/detectFramework');
+	});
+
```

Comment:
Your test uses fake timers, but they are never reset. Would suggest cleaning that up in an `afterEach` block, maybe something like: 

```suggestion

afterEach(() => {
    jest.useRealTimers();
});
```